Jenkinsfile

```diff
@@ -4,8 +4,42 @@ properties(
   ]
 )
 
+// https://issues.jenkins-ci.org/browse/JENKINS-33511
+def set_workspace() {
+  if(env.WORKSPACE == null) {
+    env.WORKSPACE = pwd()
+  }
+}
+
 node('buildvm-devops') {
-  sshagent(['openshift-bot']) { // git repo privileges stored in Jenkins credential store
-    echo "Bruno's branch split pipeline"
+  try {
+    deleteDir()
+    set_workspace()
+    stage('clone') {
+      dir('aos-cd-jobs') {
+        checkout scm
+        sh 'git checkout master'
+      }
+    }
+    stage('run') {
+      sshagent(['openshift-bot']) { // git repo privileges stored in Jenkins credential store
+        sh '''\
+virtualenv env/
+env/bin/pip install gitpython
+export PYTHONPATH=$PWD/aos-cd-jobs
+env/bin/python aos-cd-jobs/aos_cd_jobs/pruner.py
```

Comment:
To me that's one more reason to use the binary, not the `activate` script.
From virtualenv's docs:

> This will change your `$PATH` so its first entry is the virtualenv’s `bin/`
> directory. (You have to use `source` because it changes your shell
> environment in-place.) This is all it does; it’s purely a convenience. If you
> directly run a script or the python interpreter from the virtualenv’s `bin/`
> directory (e.g. `path/to/ENV/bin/pip` or `/path/to/ENV/bin/python-script.py`)
> there’s no need for activation.

https://virtualenv.pypa.io/en/stable/userguide/#activate-script